source/android/mobilechatapp/src/main/assets/Action.OpenUrl.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "This card's action will open a URL"
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.OpenUrl",
+			"title": "Action.OpenUrl",
+			"url": "http://adaptivecards.io"
+		}
+	]
+}
```

Comment:
Can we split these into folders (Scenarios, Tests, Elements) and have a step that copies them from the \Samples location. Having multiple copies is just going to lead to discrepancies in the the tests between this renderer and the other renderers.